src/api/app/models/comment.rb

```diff
@@ -2,16 +2,10 @@
 require 'set'
 
 class Comment < ApplicationRecord
-  belongs_to :bs_request, inverse_of: :comments
-  belongs_to :project, inverse_of: :comments
-  belongs_to :package, inverse_of: :comments
+  belongs_to :object, polymorphic: true # belongs to a Project, Package or BsRequest
```

Comment:
`object` is super generic name. I think `commentable` is the most frequent thing people use for this.